scene/gui/tab_container.cpp

```diff
@@ -1117,26 +717,28 @@ Size2 TabContainer::get_minimum_size() const {
 		ms.y = MAX(ms.y, cms.y);
```

Comment:
The size problem probably lies here. The MAX should be **added** to `ms` from tabs, not overwrite it. So you probably need two separate sizes - one for tabs and one for controls and then add them.